### ISSUE TITLE
Inconsistent appearance on `select` css components.

### ISSUE THREAD

**Issue by anatoo** [OnsenUI/OnsenUI#2017]
__Environment__ 

```
[Core]
    current master branch

    Desktop - macOS Sierra

[Browser]
    Desktop - Safari 10.1 (12603.1.30.0.34)
```

__Encountered problem__

![image](https://cloud.githubusercontent.com/assets/59784/26194960/1d09d8fa-3bf5-11e7-949f-eb4d58073bf2.png)

**Comment by asial-matagawa** [comment]
@anatoo
This issue is also discussed in https://github.com/OnsenUI/OnsenUI/issues/1864 .
Maybe we require a lot effort to fix `select` component.

**Comment by rgins16** [comment]
@anatoo @asial-matagawa I am having the problem described in #1864. I am running 10.3.2 on my iPhone 5C.

**Comment by asial-matagawa** [comment]
@rgins16 
This problem has been fixed in #2028 and the fix will be included in the next version of `onsenui` (`2.3.2`).
Thank you for reporting!

**Comment by rgins16** [comment]
@asial-matagawa YAY! Thanks team!
